.github/container/Dockerfile.jax

```diff
@@ -47,6 +47,11 @@ ARG SRC_PATH_JAX
 ARG SRC_PATH_XLA
 ARG BUILD_DATE
 ENV BUILD_DATE=${BUILD_DATE}
+ENV XLA_FLAGS="--xla_gpu_enable_latency_hiding_scheduler=true --xla_gpu_enable_async_all_gather=true --xla_gpu_enable_async_reduce_scatter=true --xla_gpu_enable_triton_gemm=false"
```

Comment:
@abhinavgoel95 should we also include `xla_gpu_enable_pipelined_all_reduce` and `xla_gpu_enable_pipelined_all_gather` to this list?

Reply:
No, we see perf issues with those flags as of today. Hoping they get fixed by early next week.